### PR TITLE
Feature bulgarian country regions

### LINKED ISSUE #28215: [Issue] Feature bulgarian country regions
This issue is automatically created based on existing pull request: magento/magento2#27957: Feature bulgarian country regions

---------

### Description (*)
Adding Bulgarian regions to **directory_country_region** table

### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Bulgarian states.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by nikolalardev**
### Description (*)
Adding Bulgarian regions to **directory_country_region** table

### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Bulgarian states.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#28215: Feature bulgarian country regions

### CODE REVIEW

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForBulgaria.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\Setup\Patch\PatchVersionInterface;
+
+/**
+ * Add Bulgaria States
+ * 
+ * Class AddDataForBulgaria
+ */
+class AddDataForBulgaria implements DataPatchInterface, PatchVersionInterface
```
Comment (rogyar) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForBulgaria.php]: Please, remove the `PatchVersionInterface` dependency. We may omit it in this particular case and remove the redundant `getVersion()` as result.

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @nikolalardev. Thank you for your collaboration. Please, check my comments below for some minor adjustments. 

Also, according to the [definition of done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) all changes should be covered by automated tests. You may refer to the following PR that shows how to create an MFTF test for a similar case.

https://github.com/magento/magento2/pull/25590

Thank you.

**Comment by engcom-Echo** [comment]
I will take care of test coverage.

**Comment by kalinicham** [comment]
Hi @rogyar could you please approve it?

**Comment by engcom-Echo** [comment]
Hi @rogyar could you please approve it?

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-7536](https://jira.corp.magento.com/browse/ENGCOM-7536) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@nikolalardev thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

Go to Admin->Stores->Configuration->General->Store Information

**Actual Result:**
Screenshot from 2020-05-13 10-31-56

Go to Checkout process->Shipping

**Actual Result:**
Screenshot from 2020-05-13 10-35-56

**Comment by slavvka** [comment]
@magento create issue

**Pull request merged**